react/helpers/helpersTypesAccepted.spec.js

```diff
@@ -0,0 +1,83 @@
+import {
+  isValidFile,
+  isValidFolder,
+  getValidTypesAccepted
+} from './helpersTypesAccepted'
+
+const makeMockFile = ({
+  extension = '.pdf',
+  mime = 'application/pdf'
+} = {}) => ({
+  _id: '123',
+  type: 'file',
+  mime: mime,
+  name: `mockFile${extension}`
+})
+const makeMockFolder = () => ({
+  _id: '789',
+  type: 'directory',
+  name: 'mockDir'
+})
+
+describe('helpersTypesAccepted', () => {
+  describe('getValidTypesAccepted', () => {
+    it('should an array with all valid types', () => {
+      const res = getValidTypesAccepted('.pdf, text/plain, .not')
+
+      expect(res).toStrictEqual(['.pdf', 'text/plain'])
+    })
+  })
+
+  describe('isValidFile', () => {
+    it('should be valid when item has no type providen', () => {
+      const item = makeMockFile()
+      const validTypesAccepted = []
+
+      expect(isValidFile(item, validTypesAccepted)).toBe(true)
+    })
+
+    it('should be valid when item has an accepted extension', () => {
+      const item = makeMockFile({ extension: '.png' })
+      const validTypesAccepted = ['.png', '.pdf']
+
+      expect(isValidFile(item, validTypesAccepted)).toBe(true)
+    })
+
+    it('should not be valid when item has not an accepted extension', () => {
+      const item = makeMockFile({ extension: '.png' })
+      const validTypesAccepted = ['.pdf']
+
+      expect(isValidFile(item, validTypesAccepted)).toBe(false)
+    })
+
+    it('should be valid when item has an accepted mime', () => {
```

Comment:
let's add one more test with a not accepted mime 🙏

Reply:
test not added

Reply:
Sorry, I didn't push the last changes